challenges/objects-arrays.js

```diff
@@ -50,21 +80,32 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (i=0; i < graduates.length; i++) {
+  universities.push(graduates[i]["university"])
 console.log(universities)
-
+}
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (i=0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i]["first_name"]);
+  contactInfo.push(graduates[i]["email"]);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
-console.log(uni);
+for (i = 0; i < graduates.length; i++) {
+  if(graduates[i].university.includes('Uni')) {
+    uni.push(graduates[i]);
+  }
+}
+console.log(uni.length+` universities have "Uni" included in their name`);
```

Comment:
Great, but would be good to use template string literals here instead of `+ <string>`